src/ranged.cpp

```diff
@@ -815,6 +815,11 @@ int Character::fire_gun( const tripoint &target, int shots, item &gun )
         const vehicle *in_veh = has_effect( effect_on_roof ) ? veh_pointer_or_null( here.veh_at(
                                     pos() ) ) : nullptr;
 
+        // Add gunshot noise if player is the one who shot
+        const item::sound_data gunnoise = gun.gun_noise( shots > 1 );
+        add_msg_if_player( _( "You shoot your %1$s. %2$s" ), gun.tname(),
+                           uppercase_first_letter( gunnoise.sound ) );
```

Comment:
I'd remove the `You shoot your X` part.
First, I know that I'm firing right now. Second, I know what gun I'm firing right now. Having this in mind, `You shoot your X` part looks superfluous.

```suggestion
        add_msg_if_player( _( "%s" ), uppercase_first_letter( gunnoise.sound ) );
```

Reply:
It seems to me that some message telling me what I did is standard in this game.

I pick this up. I open that door. I wield that.  By that same logic, should all of those be removed?

I kind of think it should stay in that it says Im shooting in some way, along with the noise effect.

Looking at the screenshots above, the shoot message fits perfectly (in my opinion)

Reply:
Also if you shoot a silenced weapon, you won't get any noise message so you should still get some kind of message for that, I think